src/components/Users/LocationSearch.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import cities from '../../cities.json'
```

Comment:
gotta minify this JSfile, doesn't have to be in the commit though. little about minification:
[webpack production guide](https://webpack.js.org/guides/production/)